data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,319 @@ mission "Wanderers: Pre-Colony"
+	landing
+	name "Heavy machinery to <planet>"
+	description "Bring <cargo> from <stopovers> to <destination> to build a new outfitter. Payment is <payment>."
+	blocked "The building supervisor at <destination> has [ordered, requisitioned] more equipment than you can carry. You will need <capacity> to transport it."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Varu Tek'kai"
+	cargo "heavy machinery" 300
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+
+	on stopover
+		dialog "You notice that each crate of machinery has been signed by the dock team that loaded it."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort carrying the <cargo> hasn't landed. You realize you need to depart and wait for it."
+
+	on complete
+		dialog "The construction supervisor drops by as the workers are unloading. He gives you a credit chip for <payment> and asks you to meet him in the tent that serves as the spaceport when all the deliveries are done."
+		payment 250000 3
+
+
+mission "Wanderers: Spera Anatrusk Outfitter 3"
+	landing
+	name "Warehouse equipment to <planet>"
+	description "Deliver warehouse equipment and staff for the outfitter on <destination>."
+	source "Spera Anatrusk"
+	destination "Vara K'chrai"
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 2a: done"
+		has "Wanderers: Spera Anatrusk Outfitter 2b: done"
+		has "Wanderers: Spera Anatrusk Outfitter 2c: done"
+
+	on offer
+		conversation 
+			`This time you land on a concrete pad and you see the familiar hexagonal layout of a Wanderer spaceport. There are tents instead of buildings but you can see stakes outlining the shapes to come. You pass a group of Wanderer captains who appear to be holding a swap meet looking for spare ammunition and equipment so they can avoid the trip back across the Eye.`
+			`	The construction supervisor bobs his head enthusiastically when he sees you and eventually remembers to switch to Hai. "Thank you, Captian <last>," he says. "We would be waiting weeks to get started building without your help. We now need to get the warehouse equipment and [personnel, staff] in to organize the outfitter and [technicians, fixers] do do the [maintenance, repairs]. Would you be able to do this for us as well?`
+			choice
+				`	"Of course. I need an outfitter here as much as anybody."`
+					goto willdo
+				`	"You'll need to find someone else for the next part."`
+					decline
+			label willdo 
+			`	"That is [optimal, excellent]," he says as he hands you a data chip. If you will take this to <destination>, they will make all the arrangements.`
+				accept
+	
+	on complete
+		conversation
+			`Things go much smoother this time. The Wanderer in the office takes the chip and after a few moments loads their own translation application and says, "[Complete, Done]. On job [board, list] soon. Spaceport if no see."`
+			`	You thank them and wonder how strange your translator makes you sound to the Wanderers.`
```

Comment:
As a general rule, it's best to avoid telling the player how they feel. Maybe they don't care about how the translator makes them sound. Maybe the thought has never occurred to them. Maybe they have just been assuming all along that the translator is hilariously barely-competent and that the Wanderers are secretly laughing at them all the time. 

Anyway, point being that everyone's unique, and people like their pilot to be so too. So it's better to describe external stimuli that leads them to make certain conclusions for themselves.

```suggestion
			`	You thank them, and as the translator conveys it to the Wanderers, you can see them twitch their head and make noises the translator simply states as [amusement, appreciation].`
```

Noting that further down you mention them making a head motion that the player has associated with amusement. So, introducing the idea here.